woocommerce/wc-admin
# Added event prop 'screen' to be recorded

Fixes #4633

The event prop `screen` wasn't being sent with the event `wcadmin_wcadmin_inbox_action_click`.
This PR adds the event prop `screen` to `wcadmin_wcadmin_inbox_action_click`.
This change will apply for all the call to action buttons of the admin notes.


### Accessibility




### Detailed test instructions:
- Open the Inbox Panel, choose a message, and press its call to action button.
- Verify that the screen name has been sent as a prop in the event `wc_admin_record_tracks_event` when a note's action is triggered. ([Here](https://github.com/woocommerce/woocommerce-admin/blob/16770825fe26015e32f52c75a8331502638af880/src/API/NoteActions.php#L133) is where we send it).
- Go to another page (E.g. the Analytics Overview, URL ->`/wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Foverview`).
- Verify the sent screen name corresponds with the page name.
- Also, verify the screen name is sent with other types of admin notes.
E.g.:
![screenshot-one wordpress test-2020 06 19-13_17_53](https://user-images.githubusercontent.com/1314156/85155797-97e81d80-b22f-11ea-8e3e-9cdb9d32417c.png)

- If you don't have other types of admin notes available to test, you can add one executing these two sentences in the DB:
````
INSERT INTO `wp_wc_admin_notes` (`note_id`, `name`, `type`, `locale`, `title`, `content`, `content_data`, `status`, `source`, `date_created`, `date_reminder`, `is_snoozable`, `layout`, `image`, `is_deleted`, `icon`) VALUES (9999, 'wc-admin-test-note', 'update', 'en_US', 'Lorem ipsum note', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.', '{}', 'unactioned', 'woocommerce-admin', '2020-06-17 19:22:48', NULL, 0, 'plain', NULL, 0, 'info');
````
and
````
INSERT INTO `wp_wc_admin_note_actions` (`note_id`, `name`, `label`, `query`, `status`, `is_primary`, `actioned_text`) VALUES (9999, 'learn-more', 'Learn more', '?page=wc-addons&section=helper', 'unactioned', 0, '');
````



### Changelog Note:
Fix: Added event prop `screen` to `wcadmin_wcadmin_inbox_action_click`.
